src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.Android.cs

```diff
@@ -11,7 +11,7 @@ public static partial class Clipboard
 	{
 		public static void SetContent(DataPackage content)
 		{
-			if (content?.Text == null)
+			if (content?.Text != null)
```

Comment:
There's also the issue that resetting the content of the clipboard does not work.

Reply:
@jeromelaban I checked and putting null just make the app crash. But setting the text to empty reset the clipboard